src/data/reports/glass-data-submission/GLASSDataSubmissionDefaultRepository.ts

```diff
@@ -56,6 +56,13 @@ interface MessageConversations {
     }[];
 }
 
+interface TrackedEntityInstance {
```

Comment:
It could be an `interface`, but as a convention use `type` instead of `interface` except when you need some specific features of `interface` (for example for repositories)